service/frontend/wrappers/metered/metered_test.go

```diff
@@ -92,3 +94,452 @@ func TestSignalMetricHasSignalName(t *testing.T) {
 	}
 	assert.True(t, expectedMetrics["test.cadence_requests"])
 }
+
+func TestHandleErr_InternalServiceError(t *testing.T) {
+	logger := testlogger.New(t)
+	testScope := tally.NewTestScope("test", nil)
+	metricsClient := metrics.NewClient(testScope, metrics.Frontend)
+	handler := &apiHandler{}
+
+	err := handler.handleErr(&types.InternalServiceError{Message: "internal error"}, metricsClient.Scope(0), logger)
+
+	assert.NotNil(t, err)
+	assert.Contains(t, err.Error(), "cadence internal error")
+	assert.Contains(t, err.Error(), "internal error")
```

Comment:
this line is redundant given above line

Reply:
noted! removing it now and will include the change in second PR